charts/backbeat-consumer/values.yaml

```diff
@@ -4,8 +4,8 @@
 replicaCount: 1
 image:
   repository: zenko/backbeat
-  tag: pensieve-2
-  pullPolicy: IfNotPresent
+  tag: 0.1.3
+  pullPolicy: Always
```

Comment:
Why?

Reply:
I think it was a dev setting. @JianqinWang can you plz update the pullPolicy ?